core/src/main/scala/kafka/coordinator/TransactionLogManager.scala

```diff
@@ -14,7 +14,18 @@
   * See the License for the specific language governing permissions and
   * limitations under the License.
   */
-
+/**
```

Comment:
The header we want is in checkstyle/java.header. Admittedly we don't have any checks for scala headers, so it's not obligatory to change this. But if we do, we should remove the old header.